### PR TITLE
Release notes for 5.16.1

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Update release notes to add 5.16.1 release.

## Motivation and Context
Support 5.16.1 release

### CODE REVIEW

File: content/sensu-go/5.16/release-notes.md [hunk 1]
```diff
@@ -49,6 +50,18 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is a patch release that fixes a performance issue ...
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This is a patch release that optimizes the function the entity limiter uses.
```

File: content/sensu-go/5.16/release-notes.md [hunk 2]
```diff
@@ -49,6 +50,18 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is a patch release that optimizes the function the entity limiter uses.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**FIXES:**
+
+- ...
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
- Fixed a performance regression introduced in 5.15.0 that caused the API to timeout past 20k agent sessions.
```

File: content/sensu-go/5.16/release-notes.md [hunk 3]
```diff
@@ -49,6 +50,18 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is a patch release that optimizes the function the entity limiter uses.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**FIXES:**
+
+- Fixed a performance regression introduced in 5.15.0 that caused the API to timeout past 20k agent sessions.
+- ...
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
```

File: content/sensu-go/5.16/release-notes.md [hunk 4]
```diff
@@ -49,6 +50,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is a patch release that optimizes the function the entity limiter uses.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md:59]:
```suggestion

**IMPROVEMENTS**

- The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.

```

File: content/sensu-go/5.16/release-notes.md [hunk 5]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is a patch release that optimizes the function the entity limiter uses.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This is release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```

File: content/sensu-go/5.16/release-notes.md [hunk 6]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This is release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```

File: content/sensu-go/5.16/release-notes.md [hunk 7]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**IMPROVEMENTS**
+
+- The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.
+
+**FIXES:**
+
+- Fixed a performance regression introduced in 5.15.0 that caused the API to timeout past 20k agent sessions.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
- The performance regression introduced in 5.15.0 that caused the API to time out past 20k agent sessions is now fixed.
```

File: content/sensu-go/5.16/release-notes.md [hunk 8]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**IMPROVEMENTS**
+
+- The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.
```
Comment (nikkictl) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
- The `sensu_go_events_processed` prometheus counter now initializes with the `success` label so the count is always displayed.
```

File: content/sensu-go/5.16/release-notes.md [hunk 9]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```
Comment (nikkictl) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` prometheus counter by default.
```

**Review by echlebek — COMMENTED** [review]
I think it would be better not to mention 20k agents specifically. For one thing, this number was spread out over 3 servers. For another thing, it depends on the performance of the CPU used. So it's likely that some users will encounter this issue with fewer agents, and some users with more.

Let's try to frame it in terms of a scaling problem, ie, "As the number of connected agents increases, API latency scales linearly" or something like that.

**Review by nikkictl — APPROVED** [review]
Let's add some context about `sensu_go_events_processed` so folks know how to access it. It's a prometheus counter for reference.

Everything else looks good!

File: content/sensu-go/5.16/release-notes.md (hunk 9 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` counter by default.
```

File: content/sensu-go/5.16/release-notes.md [hunk 10]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**IMPROVEMENTS**
+
+- The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.
+
+**FIXES:**
+
+- The performance regression introduced in 5.15.0 that caused the API to time out past 20k agent sessions is now fixed.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
- The performance regression introduced in 5.15.0 that caused API latency to scale linearly as the number of connected agents increased is fixed.
```

File: content/sensu-go/5.16/release-notes.md [hunk 11]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` counter by default.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.
+
+**IMPROVEMENTS**
+
+- The `sensu_go_events_processed` prometheus counter now initializes with the `success` label so the count is always displayed.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
- The `sensu_go_events_processed` Prometheus counter now initializes with the `success` label so the count is always displayed.
```

**Review by echlebek — APPROVED** [review]

File: content/sensu-go/5.16/release-notes.md [hunk 12]
```diff
@@ -49,6 +50,21 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.1 release notes
+
+**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
+This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` counter by default.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/release-notes.md]:
```suggestion
This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` Prometheus counter by default.
```

**Review by nikkictl — APPROVED** [review]

**Pull request merged**